styles/base.css

```diff
@@ -1,25 +1,27 @@
-@import url("./fonts.css");
-
 :root {
```

Comment:
Preliminary performance comparison between externally-hosted and self-hosted fonts. Keep in mind, the externally-hosted fonts are request more font-weights now - 2 for Roboto and 4 for Source Code Pro (vs. 1 for Roboto and 2 for Source Code Pro). 

This PR vs. Production, ran tests 2 times in a row:

<img width="1512" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/1436fd7b-cfaf-4f57-9103-3c7074e4ea2b">
<img width="1512" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/8e80b449-d1c6-4718-b278-a558028226df">

It's hard to get consistent Performance results (like look at how Bootstrap differs across tests) - but it seems like loading 4 weights of Source Code Pro via externally-hosted is still faster than loading 2 weights of self-hosted.

Technically, we're not using weight 800 for Source Code Pro yet, but I kept it in b/c it'll be used for the H1, which is used in the About pages coming up.